fvtest/porttest/si.cpp

```diff
@@ -860,6 +860,10 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 	{
 		OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
 		const char *testName = "omrsysinfo_test_sysinfo_set_limit_FILE_DESCRIPTORS";
+#if defined(OSX)
+		portTestEnv->log("Skipping test: sysinfo_test_sysinfo_set_limit_FILE_DESCRIPTORS: resource limits unsupported on macOS, see issue #1234 for more details\n");
```

Comment:
Wait! I forgot we need to amend this. I'll open an issue by EOD.

Reply:
Oops! Will hold off.

Reply:
Ok done. @mw918 could you replace 1234 with 4990?